news/2022-10-21-project-loved-october-2022.md

```diff
@@ -0,0 +1,374 @@
+170 BPM, 4:22 | 5.61★\
+*description written by [Wesley](https://osu.ppy.sh/users/2407265)*
+
+This month we have a newcomer to the Loved category, [-Vale](https://osu.ppy.sh/users/7075339)! While this isn't the most challenging map for everyone this Loved rotation, I'm sure there's a bunch of players that will find enjoyment out of this map.\
+With a banger of a song, the map filled with streams, some longer sliders here and there and the overall clean mapping style, you're sure to have a blast playing through the map.\
+Think the map is too easy for you? Try chucking on Hardrock and see if you can back up your claim! With a circle size of 4.5 (and almost 6 with Hardrock enabled) it will give you a run for your money!
+
+[![](/wiki/shared/news/2022-10-21-project-loved-october-2022/785482.jpg)](https://osu.ppy.sh/community/forums/topics/{{TOPIC_ID}})
+
+[ExileLord - Soulless 3](https://osu.ppy.sh/beatmapsets/785482#fruits) by [Kyptoric](https://osu.ppy.sh/users/1361268)\
+255 BPM, 5:37 | 9.56★\
+*description written by [Tenshichan](https://osu.ppy.sh/users/1101600)*
+
+Here we go again. Do you remember [Soulless 4](https://osu.ppy.sh/beatmapsets/772138#fruits/1623653), that one menacing map by [Kyptoric](https://osu.ppy.sh/users/1361268) that has only a single FC despite being loved for years already? This is pretty much its younger brother. In terms of patterns and overall feel of the map Soulless 3 is very similar to Soulless 4, so if you know about Soulless 4 you know what to expect from this one. The major difference though is that Soulless 3 is less than half the length of Soulless 4 which will probably make it more doable than its predecessor. But even then, this is a ridiculous challenge for just about anyone.
+
+[![](/wiki/shared/news/2022-10-21-project-loved-october-2022/1546722.jpg)](https://osu.ppy.sh/community/forums/topics/{{TOPIC_ID}})
+
+[Seiryu - AO-Infinity](https://osu.ppy.sh/beatmapsets/1546722#fruits) by [Saikai](https://osu.ppy.sh/users/8172283)\
+191 BPM, 3:30 | 7.22★\
+*description written by [Tenshichan](https://osu.ppy.sh/users/1101600)*
+
+[Saikai](https://osu.ppy.sh/users/8172283) is known for being an insane player who is currently top 10 on the global leader boards, but apart from all the impressive scores they set regularly, they also do map quite a lot.
+
+One of those maps is *AO-Infinity*, which is overall a clean map with some unconventional patterning here and there. A major part of this map are stream patterns with lots of hyperdashes, but there are also some parts that may throw you off, especially the middle section of the map since it contains some uncommon hyperdashes paired with fast-paced patterns. The most difficult section though is near the end of the map, where the stream patterns get noticeably more difficult and also contain quite the amount of fast wiggles in between. If you are not careful enough, a slip up may end the run there.
```

Comment:
```suggestion
One of those maps is *AO-Infinity*, which is overall a clean map with some unconventional patterning here and there. A major part of this map are stream patterns with lots of hyperdashes, but there are also some parts that may throw you off, especially the middle section of the map since it contains some uncommon hyperdashes paired with fast-paced patterns. The most difficult section though is near the end of the map, where the stream patterns get noticeably more difficult and also contain quite the amount of fast wiggles in between. If you are not careful enough, a slip-up may end the run there.
```